app/src/main/java/com/lee989898/soptlee/signup/SignUpViewModel.kt

```diff
@@ -0,0 +1,33 @@
+package com.lee989898.soptlee.signup
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+enum class NoticeType{
+    NAME, ID, PWD
+}
+
+class SignUpViewModel : ViewModel() {
+
+    private val _notice = MutableLiveData<String>()
+
+    val notice: LiveData<String>
+        get() = _notice
+
+    init {
+        _notice.value = ""
+    }
```

Comment:
```suggestion
    private val _notice = MutableLiveData("")
    val notice: LiveData<String> = _notice
```

`get()`의 사용과 관련한 [질문](https://github.com/30th-WE-SOPT-Android-Part/Android-HyoRim/pull/1#discussion_r850206888)과 그에 대한 [답변](https://github.com/30th-WE-SOPT-Android-Part/Android-HyoRim/pull/1#issuecomment-1099875246) 내용 공유해 드립니다~!!